src/io/flutter/sdk/FlutterSdkUtil.java

```diff
@@ -186,7 +186,6 @@ public static String getErrorMessageIfWrongSdkRootPath(final @NotNull String sdk
 
   public static boolean isFlutterProjectDir(@Nullable VirtualFile dir) {
     if (dir == null || !dir.isDirectory()) return false;
-    if (dir.findChild(FlutterConstants.FLUTTER_YAML) != null) return true;
```

Comment:
Same here re: removing the check; we may want to keep but remove after a certain time.